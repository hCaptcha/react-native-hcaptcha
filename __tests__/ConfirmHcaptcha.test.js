import React from 'react';
import renderer from 'react-test-renderer';
import ConfirmHcaptcha from '../index';

describe('ConfirmHcaptcha snapshot tests', () => {
  it('renders ConfirmHcaptcha with minimum props', () => {
    const component = renderer.create(
      <ConfirmHcaptcha
        siteKey="00000000-0000-0000-0000-000000000000"
        baseUrl="https://hcaptcha.com"
        languageCode="en"
      />
    );
    expect(component).toMatchSnapshot();
  });

  it('renders ConfirmHcaptcha with all props', () => {
    const component = renderer.create(
      <ConfirmHcaptcha
        size="compact"
        siteKey="00000000-0000-0000-0000-000000000000"
        baseUrl="https://hcaptcha.com"
        languageCode="en"
        showLoading={false}
        loadingIndicatorColor="#999999"
        backgroundColor="rgba(0.1, 0.1, 0.1, 0.4)"
        theme="light"
        rqdata='{"some": "data"}'
        sentry={true}
        jsSrc="https://all.props/api-endpoint"
        endpoint="https://all.props/endpoint"
        reportapi="https://all.props/reportapi"
        assethost="https://all.props/assethost"
        imghost="https://all.props/imghost"
        host="all-props-host"
      />
    );
    expect(component).toMatchSnapshot();
  });
});
