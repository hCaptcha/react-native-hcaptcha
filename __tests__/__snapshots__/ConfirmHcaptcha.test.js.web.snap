// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmHcaptcha snapshot tests renders ConfirmHcaptcha with all props 1`] = `null`;

exports[`ConfirmHcaptcha snapshot tests renders ConfirmHcaptcha with minimum props 1`] = `null`;
